frontend/src/models/comment.ts

```diff
@@ -0,0 +1,10 @@
+import User from "./user.ts";
+import Post from "./post.ts";
+
+export default interface Comment {
+    _id: string;
+    postID: Post;
```

Comment:
```suggestion
    post: Post;
```
But do we need this connection and not an array of comments in Post?